src/components/Facility/EncounterForm.tsx

```
@@ -0,0 +1,1532 @@
+            ? state.form.icu_admission_date
+            : undefined,
+        action: state.form.action,
+        review_interval: state.form.review_interval,
+        assigned_to:
+          state.form.is_telemedicine.toString() === "true"
+            ? state.form.assigned_to
+            : "",
+        special_instruction: state.form.special_instruction,
+        weight: Number(state.form.weight),
+        height: Number(state.form.height),
+        bed: bed && bed instanceof Array ? bed[0]?.id : bed?.id,
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)